cart/src/main/java/com/yas/cart/model/AbstractAuditEntity.java

```diff
@@ -10,6 +13,10 @@
 @Getter
 @Setter
 public class AbstractAuditEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
```

Comment:
Hi @DinhThaiVV , AbstractAuditEntity should contains only audit fields. I think we should not move Id to this abstract

Reply:
Thanks @khanhtrand 
I will close this PR and close issue too